paddlenlp/transformers/gpt/tokenizer.py

```diff
@@ -246,6 +246,18 @@ def vocab_size(self):
 
         """
         return len(self.sp)
+    
+    def get_vocab(self):
+        """
+        Returns the vocabulary as a dictionary of token to index.
+
+        `tokenizer.get_vocab()[token]` is equivalent to `tokenizer.convert_tokens_to_ids(token)` when `token` is in the
+        vocab.
+
+        Returns:
+            `Dict[str, int]`: The vocabulary.
+        """
+        return {self.sp.IdToPiece(i): i for i in range(0, self.sp.GetPieceSize())}
```

Comment:
非常感谢解决修复此问题，不过这里还需要考虑：`added_tokens_encoder`，建议调整为以下代码：

```suggestion
        return dict({self.sp.IdToPiece(i): i for i in range(0, self.sp.GetPieceSize())}, **self.added_tokens_encoder)
```

具体可参考：

https://github.com/PaddlePaddle/PaddleNLP/blob/7d48f0eba89056cebbe3d3745f40112c0e9ac638/paddlenlp/transformers/bert/tokenizer.py#L452-L453